app/main.py

```diff
@@ -1,8 +1,37 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: str, clean_mark: str, brand: str)\
+            -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: int, clean_power: int,
+                 average_rating: int, count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(self, car: str) -> None:
```

Comment:
The `calculate_washing_price` method has an incorrect type hint for the `car` parameter. It should be `Car` instead of `str`, as this method is supposed to work with instances of the `Car` class.